test/arbitrage/BancorArbitrage.ts

```diff
@@ -986,9 +1067,6 @@ describe('BancorArbitrage', () => {
         const uniV3Fees = [100, 500, 3000];
         const tokenSymbols = [TokenSymbol.TKN1, TokenSymbol.TKN2, TokenSymbol.ETH];
 
-        // remove BancorV3 exchange until the reentrancy guard issue is resolved
-        exchangeIds.splice(exchangeIds.indexOf(ExchangeId.BancorV3), 1);
```

Comment:
exists in other places in the test file as well

Reply:
Fixed - arb tests now use all exchanges.